docs/releases/1.9.txt

```diff
@@ -416,6 +419,11 @@ Templates
 * The ``include`` tag now caches parsed templates objects during template
   rendering, speeding up reuse in places such as for loops.
 
+* Support for iterating over
+  :class:`~django.template.backends.django.DjangoTemplates` was added. This
```

Comment:
this link is confusing as we are not actually iterating over the `DjangoTemplates` backend.